projects/crc-timing/testing/test_crc_timing.py

```
@@ -0,0 +1,317 @@
+            # waits until propery crc-custom.target is active
+            # `systemctl wait` not available yet in RHCOS
+            "time sudo systemd-run --wait --unit=topsail-wait-system-ready --property=After=crc-custom.target /bin/true; ret=$?; [[ $ret == 0 ]] && echo SUCCESS || echo FAILED; exit $ret",
+            capture_stdout=True
+        ).stdout
+        elapsed = time.time() - start_time
+        if "SUCCESS" in out:
+            logging.info(f"wait_vm_crc_ready: ready after {elapsed}s")
+            return
+        elif "FAILED" in out:
+            logging.info(f"wait_vm_crc_ready: FAILED after {elapsed}s")
+            raise RuntimeException("'crc-custom.target' failed to get ready")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)